R/mlg.r

```diff
@@ -82,7 +82,9 @@
 #'   
 #' @return \subsection{mlg}{ an integer describing the number of multilocus
 #' genotypes observed. } \subsection{mlg.table}{ a matrix with columns
-#' indicating unique multilocus genotypes and rows indicating populations. } 
+#' indicating unique multilocus genotypes and rows indicating populations. The
+#' diversity metrics are the Shannon-Weaver index (H), Stoddart and Taylor's 
+#' index (aka inverse Simpson's index; G), Simpson's index (lambda) and evenness (E5).} 
```

Comment:
Perhaps: 

```diff
-The diversity metrics are
+This table can be used with the funciton \code{\link{diversity_stats}} to calculate
```

Also, you seemed to have missed the oxford comma: "Simpson's index (lambda)**, and** evenness